pages/invitation/index.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/**
+ * @file pages/submission/index.php
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2003-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @ingroup pages_submission
+ *
+ * @brief Handle requests for the submission wizard.
+ *
+ */
+
+switch ($op) {
+    case 'decline':
+    case 'accept':
+        define('HANDLER_CLASS', 'PKP\pages\invitation\PKPInvitationHandler');
```

Comment:
The use of `HANDLER_CLASS` is deprecated; the page index wrappers can just return the handler class directly. See e.g.: https://github.com/pkp/ojs/blob/main/pages/about/index.php

Reply:
Fixed.